src/parsers/manifestjson.js

```diff
@@ -193,6 +194,19 @@ export default class ManifestJSONParser extends JSONParser {
 
     if (error.keyword === 'required') {
       baseObject = messages.MANIFEST_FIELD_REQUIRED;
+    } else if (error.keyword === 'deprecated') {
+      if (
+        this.isStaticTheme &&
+        DEPRECATED_STATIC_THEME_LWT_ALIASES.includes(error.dataPath)
+      ) {
+        baseObject = messages.MANIFEST_THEME_LWT_ALIAS;
+        // Overwrite the message with the shorter one included in the linter messages.
+        overrides.message = baseObject.message;
+      }
+      // TODO(#2462): add a messages.MANIFEST_FIELD_DEPRECATED and ensure that deprecated
+      // properties are handled properly (e.g. we should also detect when the deprecated
+      // keyword is actually used to warn the developer of additional properties not
+      // explicitly defined in the schemas).
```

Comment:
tbh, if we have an issue filed we might as well just move that comment there and remove it here. No one really will read it in the code anyway I think.